egs/aesrc2020/asr1/local/files/cvlist/UK_cv_spk

```diff
@@ -0,0 +1,8 @@
+UK-G00025
```

Comment:
Can we get the split from somewhere else?

Reply:
I took this from the challenge directly. I thought it would be best to leave such a split unchanged - what do you think?

Reply:
Sounds good!